src/swapnaWJun20/FibonacciSeries.java

```diff
@@ -0,0 +1,22 @@
+package swapnaWJun20;
+
+public class FibonacciSeries {
+	public static void main(String[] args) {
+		FibonacciSeries fibonacciSeries = new FibonacciSeries();
+		fibonacciSeries.calculateFibonacciSeries(5);
+	}
+
+	private void calculateFibonacciSeries(int input) {
+		System.out.println("Fibonacci Series for " + input + " numbers : ");
+		int first = 0;
+		int second = 1;
+		int result = 0;
+
+		for (int index = 0; index < input; index++) {
+			result = first + second;
```

Comment:
Hi Swapna
I got the number in the input requirement wrong. 
There is one change in the requirement which says for 
Program 2: To print first n Fibonacci Numbers.
Input n = 6
Output = 1 1 2  3  5  8
So Please make the condition index<input-1 in the for loop to make the necessary changes and print the desired numbers. Please make the changes and i am approving the code.